include/boost/json/impl/pointer.ipp

```diff
@@ -25,7 +24,8 @@ public:
     pointer_token(
         string_view::const_iterator b,
         string_view::const_iterator e) noexcept
-        : b_(b), e_(e)
+        : b_(b)
+        , e_(e)
```

Comment:
Note that this isn't just style... putting each statement on its own line makes it easier to get a breakpoint, and see coverage.